### ISSUE TITLE
full switch module report output

### ISSUE THREAD

**Issue by pisuke** [faucetsdn/daq#428]
The report section of the switch module must include the output of the switch commands.

For reference, here is a report showing that the output from the switch commands is empty in the report section.

[device_report_poe_device.pdf](https://github.com/faucetsdn/daq/files/4651654/device_report_poe_device.pdf)

**Comment by jhughesbiot** [comment]
Is there a clear example and/or documentation of what is expected from the framework for it to populate this information?  Are any modules using this fully?

Right now that module uses:

```
 write_out_monolog $REPORT \
                      $MANIFEST \
                      $MONO_LOG \
                      $RESULT_LINES
```

But all of those have data in them so I need some feedback on what is actually expected for this data to show up as I can't see what is missing.

**Comment by grafnu** [comment]
The misc/test_ping file has a reasonable use, is that sufficient? If not,
what exactly are you looking for?

The test_tls also has a reasonable use/example... but I'm pretty sure you
wrote that.  So maybe I don't understand your question.  I don't understand
what you mean by "But all of those have data in them" -- can you try
restating your question?

**Comment by jhughesbiot** [comment]
I spent time on the tests and their results for the TLS module but really didn't touch the framework as much as I changed the tests themselves.  

From what I can tell, they both use the exact same method to generate data to the report, have the same method to generate the report section but the switch module doesn't show up anything and the tls module does.

Managed to figure out that the report.txt being created contains everything, logs, errors and results and mechanisms outside of the module seem to be handling all the processing of that data to the right location.  Working on a PR now to add the necessary logging and should have that ready soon.
